src/main/java/TaskList.java

```diff
@@ -1,4 +1,75 @@
-package PACKAGE_NAME;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
 
 public class TaskList {
+
+    private static DateTimeFormatter dukeDateTimeFormatter = DateTimeFormatter.ofPattern("d/M/yyyy HHmm");
+    private ArrayList<Task> dukeTaskList = null;
```

Comment:
Not necessary to declare it as null in the variables filed (I think??) Please correct me if I'm wrong!! (Also this is not a very constructive feedback, I'm just curious whether it's necessary HAHA)

Reply:
Yes, you are right, thanks for pointing out! Instance and Class variables do not need to take a null value when they are declared. Local variables (Those declared inside methods) however, need to take a null value when first declared, from what I understand. Thank you :)